Mathlib/Topology/Separation.lean

```diff
@@ -2772,3 +2772,7 @@ instance ConnectedComponents.t2 [T2Space X] [CompactSpace X] : T2Space (Connecte
   refine ⟨Vᶜ, V, hU.compl.isOpen, hU.isOpen, ?_, hb mem_connectedComponent, disjoint_compl_left⟩
   exact fun h => flip Set.Nonempty.ne_empty ha ⟨a, mem_connectedComponent, h⟩
 #align connected_components.t2 ConnectedComponents.t2
+
+end T25
```

Comment:
I presume [line 2329](https://github.com/leanprover-community/mathlib4/blob/fa5693f48715f7619eabbf4b6cc1fe3e7d742253/Mathlib/Topology/Separation.lean#L2329) should be deleted instead of closing this second section.

Reply:
Update: it should be "end T25" instead.

Reply:
Fixed in #14620.